about.html

```diff
@@ -2,72 +2,93 @@
 
 <html lang="en">
 <head>
+  <!-- Meta informations -->
   <meta charset="utf-8">
-
-  <title>Sprint Challenge - About</title>
-
+  <!-- Google fonts -->
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+  <!-- stylesheets -->
   <link rel="stylesheet" href="css/index.css">
 
-</head>
+  <title>Sprint Challenge - About</title>
 
+</head>
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-        
-      <footer>
-        <nav>
-          <a href="index.html">Home</a>
-          <a href="#">About</a>
-          <a href="#">Products</a>
-          <a href="#">Blog</a>
-          <a href="#">Contact</a>
-        </nav>
-      </footer>
-    </div><!-- container -->        
+  <!-- header -->
+  <header>
+    <section class="flexbetween">
```

Comment:
This is perfectly fine how you have this done.  One thing you could do is remove the <section class="flexbetween"> and just use the nav and move the image into the nav.